### PR TITLE
[Tests-only] Get content-length response header of an image and a pdf

### PR DESCRIPTION
**Pull request by haribhandari07**
## Description

This PR adds acceptance test to get `Content-Length` header of an image and a pdf file.
## Related Issue

- https://github.com/owncloud/ocis/issues/296

## How Has This Been Tested?

- https://github.com/owncloud/ocis-reva/pull/234
- https://github.com/owncloud/ocis/pull/297
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/apiWebdavOperations/downloadFile.feature [hunk 1]
```diff
@@ -92,3 +92,59 @@ Feature: download file
     When user "Alice" gets the size of file "test-file.txt" using the WebDAV API
     Then the HTTP status code should be "207"
     And the size of the file should be "19"
+
+  @issue-ocis-296 @skipOnOcis
+  Scenario Outline: Get the content-length response header of a pdf file
+    Given using <dav_version> DAV path
+    Given user "Alice" has uploaded file "filesForUpload/simple.pdf" to "/simple.pdf"
```
Comment (phil-davis) [inline tests/acceptance/features/apiWebdavOperations/downloadFile.feature]:
```suggestion
    And user "Alice" has uploaded file "filesForUpload/simple.pdf" to "/simple.pdf"
```

File: tests/acceptance/features/apiWebdavOperations/downloadFile.feature [hunk 2]
```diff
@@ -92,3 +92,59 @@ Feature: download file
...
+    When user "Alice" downloads file "/simple.pdf" using the WebDAV API
+    Then the following headers should be set
+      | header           | value       |
+      | Content-Length   |  <length>   |
+    Examples:
+      | dav_version | length |
+      | old         | 9622   |
+      | new         | 225    |
```
Comment (phil-davis) [inline tests/acceptance/features/apiWebdavOperations/downloadFile.feature:107]: That is strange - why is `Content-Length` different for old and new?

File: tests/acceptance/features/apiWebdavOperations/downloadFile.feature [hunk 3]
```diff
@@ -92,3 +92,57 @@ Feature: download file
...
     And the size of the file should be "19"
+
+  @issue-ocis-296 @skipOnOcis
+  Scenario Outline: Get the content-length response header of a pdf file
+    Given using <dav_version> DAV path
+    And user "Alice" has uploaded file "filesForUpload/simple.pdf" to "/simple.pdf"
+    When user "Alice" downloads file "/simple.pdf" using the WebDAV API
+    Then the following headers should be set
+      | header           | value       |
+      | Content-Length   |  <length>   |
```
Comment (phil-davis) [inline tests/acceptance/features/apiWebdavOperations/downloadFile.feature:103]:
```suggestion
      | Content-Length   | <length>    |
```

File: tests/acceptance/features/apiWebdavOperations/downloadFile.feature [hunk 4]
```diff
@@ -92,3 +92,57 @@ Feature: download file
...
+    Given using <dav_version> DAV path
+    And user "Alice" has uploaded file "filesForUpload/testavatar.png" to "/testavatar.png"
+    When user "Alice" downloads file "/testavatar.png" using the WebDAV API
+    Then the following headers should be set
+      | header             | value     |
+      | Content-Length     | <length>  |
+    Examples:
+      | dav_version | length |
+      | old         | 35323  |
+      | new         | 225    |
```
Comment (phil-davis) [inline tests/acceptance/features/apiWebdavOperations/downloadFile.feature:134]: That is strange - why is `Content-Length` different for old and new?

Comment (HariBhandari07) [reply tests/acceptance/features/apiWebdavOperations/downloadFile.feature:134]: @phil-davis , I am getting different `Content-Length` when running test locally.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**